ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -7724,6 +7838,7 @@ static bool ggml_backend_vk_device_supports_op(ggml_backend_dev_t dev, const ggm
         case GGML_OP_IM2COL:
         case GGML_OP_TIMESTEP_EMBEDDING:
         case GGML_OP_POOL_2D:
+        case GGML_OP_RWKV_WKV6:
```

Comment:
Does this need to check `src0->type == GGML_TYPE_F32 && dst->type == GGML_TYPE_F32` to match get_pipeline?

Reply:
I'm learning from GGML_OP_POOL_2D to write it, like GGML_OP_POOL_2D, and here is the code on 5,000 lines:
```
    case GGML_OP_RWKV_WKV6:
        if (src0->type == GGML_TYPE_F32 && dst->type == GGML_TYPE_F32) {
            return ctx->device->pipeline_rwkv_wkv6_f32;
        }
        return nullptr;
    case GGML_OP_LEAKY_RELU:
        if (src0->type == GGML_TYPE_F32 && dst->type == GGML_TYPE_F32) {
            return ctx->device->pipeline_leaky_relu_f32;
        }
        return nullptr;
```
Therefore I think there is no need in line 7841.

Reply:
Looks like only F32 is supported in ggml, so I think it's OK to not add the condition in supports_op.